### PR TITLE
Update FTS FAQs for diacritic analyzers

### PR DESCRIPTION
**Pull request by sreekanth-cb**

### CODE REVIEW

**Review by Thejas-bhat — APPROVED** [review]

File: modules/fts/pages/fts-troubleshooting.adoc [hunk 1]
```diff
@@ -244,3 +244,34 @@ curl -XPOST -H "Content-Type: application/json" -u username:password http://host
...
+
+First, the user needs to create a custom analyser with an "asciifolding" character filter like below.
+
+image::fts-custom-analyzers-asciifoldings.png[,460,align=left]
+
+And then set the new custom analyser for the field that contains the diacritic characters.
+
+image::fts-custom-diacritic-analyzer.png[,460,align=left]
+
+Now, its ready for searching.
```
Comment (abhinavdangeti) [inline modules/fts/pages/fts-troubleshooting.adoc]: Perhaps a mention here about only using **analytic queries** here would do some good.

Comment (RayOffiah) [inline modules/fts/pages/fts-troubleshooting.adoc]:
```suggestion
Now, it's ready for searching.
```

**Review by RayOffiah — CHANGES_REQUESTED** [review]
Small punctuation change

**Review by RayOffiah — APPROVED** [review]

File: modules/fts/pages/fts-troubleshooting.adoc [hunk 2]
```diff
@@ -244,3 +244,34 @@ curl -XPOST -H "Content-Type: application/json" -u username:password http://host
...
+
+image::fts-custom-analyzers-asciifoldings.png[,460,align=left]
+
+And then set the new custom analyser for the field that contains the diacritic characters.
+
+image::fts-custom-diacritic-analyzer.png[,460,align=left]
+
+Now, it's ready for searching.
+
+Please note this asciifolding character filter is available since 6.5.0 release.
```
Comment (RayOffiah) [inline modules/fts/pages/fts-troubleshooting.adoc]:
```suggestion
Please note this asciifolding character filter is available since the 6.5.0 release.
```

**Comment by jon-strabala** [comment]
@RayOffiah Sreekanth Sivasankaran has left Couchbase, I also imagine that is PR applies to 7.1 and also 7.2

**Pull request merged**